core/src/main/java/org/mule/DefaultMuleEvent.java

```diff
@@ -83,7 +86,7 @@
     /** Mutable MuleEvent state **/
     private boolean stopFurtherProcessing = false;
     private int timeout = TIMEOUT_NOT_SET_VALUE;
-    private transient ResponseOutputStream outputStream;
```

Comment:
Can you remove all the references to ResponseOutputStream from this class and deprecate it?